scripts/install_ffmpeg_cuda.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
Add `set -euo pipefail` flags  ensures that the script will exit on errors, treat unset variables as errors, and fail on any failed command in a pipeline. This helps to make your script safer.

Also consider add `set -x` flag for more detailed info about which command is executed and throws errors. You can turn it off at the end of the script by using `set +x` .

Reply:
Added. Thanks for the detailed practices!

Reply:
Also added to nltk download